target-platforms/target-platform-latest/target-platform-latest.target

```diff
@@ -15,7 +15,7 @@
 			<unit id="org.eclipse.lsp4j.jsonrpc.debug" version="0.0.0"/>
 			<unit id="com.google.gson" version="0.0.0"/>
 			<unit id="com.google.guava" version="0.0.0"/>
-			<repository location="https://download.eclipse.org/lsp4j/updates/releases/0.20.1/"/>
+			<repository location="https://download.eclipse.org/lsp4j/updates/releases/0.21.0/"/>
```

Comment:
depending what you do in your codebase, you can remove xbase (and guava) now

Reply:
Awesone, thanks for the heads up!

Reply:
looks like guava is still used in a few places in lsp4e codebase